R/font.R

```diff
@@ -0,0 +1,80 @@
+#' @title Font
+#' @docType class
+#' @description
+#' Font class defining properties of a text (size, color, font)
+#' @include utils.R
+#' @export
+Font <- R6::R6Class(
+  "Font",
+  public = list(
+    size = NULL,

```

Comment:
@pchelle This is how I believe you can create your properties and only define the default value in the constructor
Or did you have issues with this approach?